src/beautifiers/beautysh.coffee

```diff
@@ -0,0 +1,15 @@
+"use strict"
+Beautifier = require('./beautifier')
+
+module.exports = class BashBeautify extends Beautifier
+  name: "beautysh"
+  link: "https://github.com/bemeurer/beautysh"
+  options: {
+    Bash:
+      indent_size: true
```

Comment:
This is extremely weird. At least the options (`src/options.json`) are correctly built now. `coffee docs/` should load Atom-Beautify and generate `docs/options.md`. Maybe look into  `docs/index.coffee` and see if it is breaking somewhere when you run `coffee docs/`?